imports/ui/stylesheets/components/forms.less

```diff
@@ -13,6 +20,18 @@
     position: relative;
     width: 100%;
 
+    &.dynamic {
```

Comment:
This allows for the following interaction:

![passwordlabel](https://user-images.githubusercontent.com/7697924/32531699-5b53b7f6-c413-11e7-96c8-97212ce67afc.gif)